backend/council_stream.py

```
@@ -327,6 +328,20 @@ async def run_council_pipeline(
+        # storage failure never breaks the streaming response.
+        try:
+            recorded = record_stage2_matches(
+                stage2_results, label_to_model,
+                conversation_id=input.conversation_id,
+                user_id=input.user_id,
+            )
+            logger.info(
+                "Recorded %d pairwise matches for conversation %s",
+                recorded, input.conversation_id,
+            )
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)